### PR TITLE
add color flag

### PR DESCRIPTION
**Pull request by viveksahu26**
Add flag for color output.
`go run main.go compliance -f --color=false ../lynk-api/scripts/samples/misc/ascii-boxes-sbom-cdx.json`

Command help:
```bash
$ go run main.go compliance -h
Check if your SBOM complies with various SBOM standards like NTIA minimum elements, BSI TR-03183-2, OpenChain Telco.
	Generate a compliance report for an SBOM file.

Usage:
  sbomqs compliance <sbom file> [flags]

Examples:
 sbomqs compliance --bsi|--oct  [--basic|--json] <SBOM file>

  # Check a BSI TR-03183-2 v1.1 compliance against a SBOM in a table output
  sbomqs compliance --bsi samples/sbomqs-spdx-syft.json

  # Check a BSI TR-03183-2 v1.1 compliance against a SBOM in a JSON output
  sbomqs compliance --bsi --json samples/sbomqs-spdx-syft.json

  # Check a OpenChain Telco compliance against a SBOM in a table output
  sbomqs compliance --oct samples/sbomqs-spdx-syft.json

  # Check a OpenChain Telco compliance against a SBOM in a JSON output
  sbomqs compliance --oct --json samples/sbomqs-spdx-syft.json

  # Check a V3 Framing document compliance  against a SBOM in a table output
  sbomqs compliance --fsct <sbom>

  # Check a V3 Framing document compliance  against a SBOM in a JSON output
  sbomqs compliance --fsct -j <sbom>


Flags:
  -b, --basic      output in basic format
  -c, --bsi        BSI TR-03183-2 v1.1 compliance
  -l, --color      output in colorful
  -D, --debug      enable debug logging
  -d, --detailed   output in detailed format
  -f, --fsct       V3 Framing document compliance
  -h, --help       help for compliance
  -j, --json       output in json format
  -n, --ntia       check for NTIA minimum elements compliance
  -t, --oct        OpenChainTelco compliance

```

### CODE REVIEW

File: cmd/compliance.go [hunk 1]
```diff
@@ -97,9 +98,12 @@ func init() {
 	complianceCmd.Flags().BoolP("debug", "D", false, "debug logging")
 
 	// Output control
-	complianceCmd.Flags().BoolP("json", "j", false, "json format")
-	complianceCmd.Flags().BoolP("basic", "b", false, "basic format")
-	complianceCmd.Flags().BoolP("detailed", "d", false, "detailed format")
+	complianceCmd.Flags().BoolP("json", "j", false, "output in json format")
+	complianceCmd.Flags().BoolP("basic", "b", false, "output in basic format")
+	complianceCmd.Flags().BoolP("detailed", "d", false, "output in detailed format")
+	complianceCmd.Flags().BoolP("color", "l", false, "output in colorful")
```
Comment (riteshnoronha) [inline cmd/compliance.go:104]: `color mode on`

Comment (riteshnoronha) [reply cmd/compliance.go:104]: this should always default to off.

**Review by riteshnoronha — CHANGES_REQUESTED** [review]

File: cmd/compliance.go (hunk 1 above)
Comment (viveksahu26) [reply cmd/compliance.go:104]: Our motive is to make it off until and unless explicity it is turned on via `--color=true` or `-l`

Comment (viveksahu26) [reply cmd/compliance.go:104]: Do we need to make it `on` or `off`, by default ?

Comment (riteshnoronha) [reply cmd/compliance.go:104]: off

Comment (viveksahu26) [reply cmd/compliance.go:104]: so, yeah that's what parameter (false) signifies to.

**Pull request merged**